linter_exclusions.yml

```diff
@@ -911,6 +916,9 @@ backup vault backup-properties set:
     hybrid_backup_security_features:
       rule_exclusions:
       - option_length_too_long
+    azure_monitor_alerts_for_job_failures:
+      rule_exclusions:
+      - option_length_too_long
```

Comment:
Since we have added a short option alias, I believe we no longer need to modify this file